Library/Homebrew/manpages/brew-cask.1.md

```diff
@@ -19,7 +19,7 @@ names, and other aspects of this manual are still subject to change.
 
 ## FREQUENTLY USED COMMANDS
 
-  * `install` [--force] [--skip-cask-deps] [--require-sha] <token> [ <token> ... ]:
+  * `install` [--force] [--skip-cask-deps] [--require-sha] [--language=&lt;iso-language&gt;[,&lt;iso-language&gt; ... ]] <token> [ <token> ... ]:
```

Comment:
Why are you using `&lt;` and others instead of the literal characters? It’s inconsistent with the rest of the document.

Reply:
I can take them out. I was going off this feature request linked in the original issue (https://github.com/caskroom/homebrew-cask/issues/31094).

It had the `<>`, and when the manpage is rendered via brew man, it does not render those symbols. Adding the &lt; &gt; makes them render correctly.

Reply:
Seems to be rendered correctly, by underlining them. Please do change to `<>`, to be in line with the rest of the document.